### DISCUSSION TITLE
Broken breakpoints in safari browser

### DISCUSSION THREAD

**Discussion by Oskii2311** [tailwindlabs/tailwindcss#12129] [Help]
When y-scroll appears in safari, it breaks the breakpoints. In other browsers (chrome, firefox) it works as expected. Do you have any ideas?

Example:
- open codesandbox in safari: https://codesandbox.io/s/bold-mendeleev-7l54px?file=/pages/index.tsx
- set the browser width to 768px
- everything is displayed
- uncomment div in index.tsx
- text disappears

![Screenshot 2023-10-03 at 11 30 51](https://github.com/tailwindlabs/tailwindcss/assets/22234674/4cbbf5bc-926c-4e6a-9088-5a39df464ac5)

![Screenshot 2023-10-03 at 11 30 41](https://github.com/tailwindlabs/tailwindcss/assets/22234674/31afec3d-ae28-4305-899a-a1d73bb295aa)

the same steps in chrome:
- open codesandbox in chrome: https://codesandbox.io/s/bold-mendeleev-7l54px?file=/pages/index.tsx
- set the browser width to 768px
- everything is displayed
- uncomment div in index.tsx
- text is still visible

![Screenshot 2023-10-03 at 11 39 25](https://github.com/tailwindlabs/tailwindcss/assets/22234674/ea20c774-ffe9-4607-a9aa-090697b1cfa3)

![Screenshot 2023-10-03 at 11 39 07](https://github.com/tailwindlabs/tailwindcss/assets/22234674/f8c25ac0-b5da-42c1-bbc6-fe8ad6257c55)

Thanks for your time :)

**Comment by alex-krasikau** [comment]
I believe that this is a Codesandbox-related issue as the resulting HTML is rendered in the iframe. As your content starts overflowing vertical space, it introduces a scrollbar gutter on the side, shrinking the document width.

<img width="769" alt="image" src="https://github.com/tailwindlabs/tailwindcss/assets/97882909/806907a3-ce84-4c2f-bfa7-f05a79881703">

Yes, I agree, that Safari scrollbar behavior differs from Chrome and Mozilla, but still, it handles breakpoints correctly when opening the page on a full screen - https://7l54px.sse.codesandbox.io.

<img width="768" alt="image" src="https://github.com/tailwindlabs/tailwindcss/assets/97882909/00875300-05e2-4f19-b06b-36f6d60c9491">

**Reply by Oskii2311** [reply]
omg, safari includes the scrollbar gutter width in window.innerWidth, while other browsers don't. This is the problem.
Thanks
 
![Screenshot 2023-10-03 at 13 19 45](https://github.com/tailwindlabs/tailwindcss/assets/22234674/99b81e8d-f277-4835-8e8c-5767fb18507e)
